service/key/key.go

```diff
@@ -22,13 +38,48 @@ func KeyVaultName(customObject azuretpr.CustomObject) string {
 	return customObject.Spec.Azure.KeyVault.Name
 }
 
+// MasterSecurityGroupName returns name of the security group attached to master subnet.
+func MasterSecurityGroupName(customObject azuretpr.CustomObject) string {
```

Comment:
Let's track adding those to ARM template parameters as well.

Reply:
Can we add this in TODO?

I can do this as separate PR, because it's quite big change.

Reply:
Yes, TODO is ok.